draft-ietf-rats-corim.md

```diff
@@ -535,23 +538,55 @@ The following describes each child element of this type.
 {::include cddl/protected-corim-header-map.cddl}
 ~~~
 
-The CoRIM protected header map uses some common COSE header parameters plus an additional `corim-meta` parameter.
+The CoRIM protected header map uses some common COSE header parameters plus additional metadata.
+Additional metadata can either be carried in a `CWT_Claims` (index: 15) parameter as defined by {{-CWT_CLAIMS_COSE}},
+or in a `corim-meta` map as a legacy alternative, described in {{sec-corim-meta}}.
+
 The following describes each child item of this map.
 
 * `alg` (index 1): An integer that identifies a signature algorithm.
 
 * `content-type` (index 3): A string that represents the "MIME Content type" carried in the CoRIM payload.
 
+At least one of:
+
+* `CWT-Claims` (index 15): A map that contains metadata associated with a signed CoRIM.
+  Described in {{-CWT_CLAIMS_COSE}}.
+
 * `corim-meta` (index 8): A map that contains metadata associated with a signed CoRIM.
-  Described in {{sec-corim-meta}}.
+  Described in {{sec-corim-meta}}. This is supported for compatibility with legacy
+  documents, but `CWT-Claims` is recommended for new documents.
+
+Documents MAY include both `CWT-Claims` and `corim-meta` to ease transition for verifiers, in which case the signer MUST ensure that their contents are semantically identical: the `CWT-Claims` issuer (`iss`) MUST have the same value as `signer-name` in `corim-meta`, and the `nbf` and `exp` values in the `CWT-Claims` MUST match the `signature-validity` in `corim-meta`.
 
 Additional data can be included in the COSE header map as per ({{Section 3 of -cose}}).
 
-### Meta Map {#sec-corim-meta}
+### CWT Claims {#cwt-claims}
+
+The CWT Claims ({{-CWT_CLAIMS_COSE}}) map identifies the entity that created and signed the CoRIM.
+This ensures the consumer is able to identify credentials used to authenticate its signer.
```

Comment:
```suggestion
This ensures the consumer is able to identify credentials used to authenticate its signer.
The CWT Claims map SHOULD NOT be used to carry any claims than identifying claims, to avoid any possible ambiguity with the CoRIM tags contained in the payload.
```

Reply:
@henkbirkholz @yogeshbdeshpande please review the above proposed change.